packages/service-library/src/servicelib/redis.py

```diff
@@ -59,16 +60,19 @@ def __post_init__(self):
                 redis.exceptions.ConnectionError,
                 redis.exceptions.TimeoutError,
             ],
+            socket_timeout=_DEFAULT_SOCKET_TIMEOUT.total_seconds(),
+            socket_connect_timeout=_DEFAULT_SOCKET_TIMEOUT.total_seconds(),
+            retry_on_timeout=True,
```

Comment:
question: is there not already a retry around that client? or what is this retry_on_timeout doing that tenacity doesn't do?

Reply:
ahh I see we already gave retry_on timeout error, this can be omitted